videoprocess/vpp3dlut.cpp

```diff
@@ -1236,10 +1236,6 @@ video_frame_process_scaling(VASurfaceID in_surface_id,
         vaDestroyBuffer(va_dpy, pipeline_param_buf_id);
     }
 
-    if (filter_param_buf_id != VA_INVALID_ID) {
```

Comment:
where to destroy filter_param_buf_id?

Reply:
triage the deadcode error, this code always not be entered, so remove it.